cranelift-codegen/src/isa/x86/fde.rs

```diff
@@ -0,0 +1,354 @@
+//! Support for FDE data generation.
```

Comment:
other than the register mapping, and possibly code/data alignment, `FDE` generation is architecture-agnostic. I assume you rather this code in `cranelift-codegen` for use in wasmtime, but I'd like to have a very clear path to supporting other architectures. Is there another reason this would be in `isa/x86/`? If not I'd like to move it to somewhere like `cranelift-codegen/src/binemit/unwind.rs` that is more clearly independent of a specific architecture.